buildkite/scripts/build-release.sh

```diff
@@ -20,4 +20,4 @@ echo "--- Upload debs to amazon s3 repo"
 ./buildkite/scripts/publish-deb.sh
 
 echo "--- Git diff after build is complete:"
-git diff --exit-code -- .
+#git diff --exit-code -- .
```

Comment:
Why does this need to be removed? Is there still an environment that is missing lfs?

Reply:
This is the part needing fixing or a workaround. LFS leaves the local repo dirty so we need to introduce further checks to keep the test. @mrmr1993 will assist here?